static/scss/enterprise.scss

```diff
@@ -0,0 +1,193 @@
+// sass-lint:disable mixins-before-declarations
+
+.enterprise-page {
+  font-family: Rajdhani;
+}
+
+.nav-line {
+  background: var(--ColorLine, linear-gradient(270deg, #f0ba46 0%, #95bb83 51.02%, #3dbdbf 100%));
+  height: 2px;
+  max-width: 100%;
+}
+
+.section-line {
+  background: var(--ColorLine, linear-gradient(270deg, #f0ba46 0%, #95bb83 51.02%, #3dbdbf 100%));
+  height: 5px;
+  max-width: 100%;
+}
+
+.enterprise-banner-block {
+  width: 100%;
+  flex-shrink: 0;
+  text-align: center;
+  padding-bottom: 60px;
+
+  @include media-breakpoint-down (sm) {
+    padding-bottom: 30px;
+  }
+}
+
+.enterprise-banner {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 50px 0px;
+
+  @include media-breakpoint-down (sm) {
+    display: block;
+    padding: 16px 5px;
+  }
+
+  img {
+    width: 544px;
+    height: 444px;
+
+    @include media-breakpoint-down (sm) {
+      width: 100%;
+      height: auto;
+      padding-top: 13px;
+    }
+  }
+}
+
+.banner-content {
```

Comment:
Please use nested styles. Some of the class names are too generic that they could conflict with other places.